variables.tf

```
@@ -2,7 +2,7 @@
 # Common variables
 ##############################################################################
 
-variable "resource_group_id" {
+variable "resource_gRoup_id" {
```

[Naming - Blocking] Variable uses mixed case (gRoup) instead of snake_case. This violates Terraform conventions and breaks all existing module usage.

```suggestion
variable "resource_group_id" {
```